goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/ViewDeed.java

```diff
@@ -20,6 +23,17 @@ public void onCreate(Bundle savedInstanceState) {
         subject.setText(deed.getSubject());
         TextView description = findViewById(R.id.deedDescription);
         description.setText(deed.getDescription());
+        View button = (Button) findViewById(R.id.edit_offer);
+        if (!deedController.isMyActiveDeedHandler()) {
+            button.setVisibility(View.GONE); }
 
     }
+
```

Comment:
you maybe should use code formatter and maybe the on save-plugin we talked about at the start of the course. You have lots of spaces and stuff thats not necessary